containers/bastion/cockpit-auth-ssh-key

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env python
```

Comment:
Is it easy to make this python3? That way we don't have to port later. If not, no worries.